Probesys/bileto
# Determine the default user's organization from their authorizations

## Related issue(s)


https://github.com/Probesys/bileto/issues/747

## How to test manually



1. Create a user
2. Verify that you cannot select an organization at this step
3. Verify that you're redirected to the authorization form
4. Give a "user authorization" to the user on a specific organization
5. Login with this user
6. Verify that you can create a ticket with this user
7. As an admin, edit the user
8. Verify that you can now select a default organization and that only one organization is selectable

## Checklist
